docs/content/reference/routing-configuration/http/middlewares/headers.md

```diff
@@ -0,0 +1,299 @@
+---
+title: "Traefik Headers Documentation"
+description: "In Traefik Proxy, the HTTP headers middleware manages the headers of requests and responses. Read the technical documentation."
+---
+
+![Headers](../../../../assets/img/middleware/headers.png)
+
+The Headers middleware manages the headers of requests and responses.
+
+A set of forwarded headers are automatically added by default. See the [FAQ](../../../../getting-started/faq.md#what-are-the-forwarded-headers-when-proxying-http-requests) for more information.
```

Comment:
Since we are in the reference section WDYT about relocating the list of headers from the FAQ into here?